README.md

```diff
@@ -0,0 +1,62 @@
+# Merkle-Trie [![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](https://www.gnu.org/licenses/agpl-3.0)
+Merkle trie used by [CodeChain](https://github.com/CodeChain-io/codechain) and [Foundry](https://github.com/CodeChain-io/foundry)
```

Comment:
The merkle trie used by [CodeChain](https://github.com/CodeChain-io/codechain) and [Foundry](https://github.com/CodeChain-io/foundry).